packages/kotti-ui/source/kotti-accordion/KtAccordion.vue

```diff
@@ -31,23 +31,25 @@ import { KottiAccordion } from './types'
 export default defineComponent<KottiAccordion.PropsInternal>({
 	name: 'KtAccordion',
 	props: makeProps(KottiAccordion.propsSchema),
-	setup(props) {
+	setup(props, { emit }) {
 		const contentInnerRef = ref<HTMLElement | null>(null)
+		const duration = 300
```

Comment:
since it's a constant, it can move outside the setup and folllow the convention of casing `ANIMATION_DURATION` or something